bin/cliain/Cargo.toml

```diff
@@ -2,6 +2,8 @@
 name = "cliain"
 version = "0.4.0"
 edition = "2021"
+license = "GNU GPL 3.0"
```

Comment:
This should be this license https://spdx.org/licenses/GPL-3.0-only.html , and the corresponding string is ` GPL-3.0-only`

Reply:
done